recipes/spans/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "spans" %}
+{% set version = "0.3.0" %}
+{% set sha256= "d17d36dc240525e97eae26ff68b63b8a9dc6581e4668581f6e189c24ea50f201" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/74/73/2d0192f38a8caad9ab439210ef943825312902cc9280c9b73a196b5be719/Spans-0.3.0.tar.gz
```

Comment:
Please use the URL from the [`example`]( https://github.com/conda-forge/staged-recipes/blob/6ca29ae558f4353a108a26f7ffb23250a021bea7/recipes/example/meta.yaml#L18 ).